src/components/menu/component.test-pw.tsx

```diff
@@ -0,0 +1,588 @@
+import React, { useState } from "react";
+import {
+  Menu,
+  MenuDivider,
+  MenuSegmentTitle,
+  MenuTitleProps,
+  ScrollableBlock,
+  MenuItem,
+  MenuWithChildren,
+  MenuFullscreen,
+  MenuFullscreenProps,
+  MenuProps,
+  MenuDividerProps,
+  ScrollableBlockProps,
+} from ".";
+import { MenuType } from "./menu.context";
+import Search from "../search";
+import GlobalHeader from "../global-header";
+import Box from "../box/box.component";
+import Typography from "../typography/typography.component";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+const menuTypes: MenuType[] = ["white", "light", "dark", "black"];
+
+export const MenuComponent = (props: Partial<MenuProps> & MenuDividerProps) => {
```

Comment:
can we delete these mock components from menu-test.stories.tsx since they are already moved to the components.test-pw.tsx?